packages/browser/src/node/plugins/pluginContext.ts

```diff
@@ -86,6 +86,7 @@ export const server = {
 export const commands = server.commands
 export const userEvent = createUserEvent(_userEventSetup)
 export { page, cdp }
+page[Symbol.for('vitest:user-event-cleanup')] = () => userEvent.cleanup()
```

Comment:
I noticed that `page.extend({ [Symbol.for("xxx")]: () => {} })` doesn't extend because `for in` doesn't iterate symbol

https://github.com/vitest-dev/vitest/blob/423b2ecf2644eddf51bbe1bd99b81335ce4e8731/packages/browser/src/client/tester/context.ts#L199-L204

But fixing `page.extend` means it'll also include `Symbol.for('vitest:component-cleanup')` like from https://github.com/vitest-dev/vitest-browser-react/blob/21e508e7c2400bea4026733c8b463212cf374304/src/index.ts#L8-L11 and that'll end up cleaning up component while it should leave the last rendered state untouched on iframe. For now, I left `page.extend` as is.

Reply:
Well, that's the point of cleanup, the symbol will have to be fixed at some point so components can be properly cleaned up